Policies/github-terms-of-service.md

```diff
@@ -332,9 +346,9 @@ Throughout this Agreement, each section includes titles and brief summaries of t
 #### 4. Severability, No Waiver, and Survival
 If any part of this Agreement is held invalid or unenforceable, that portion of the Agreement will be construed to reflect the parties’ original intent. The remaining portions will remain in full force and effect. Any failure on the part of GitHub to enforce any provision of this Agreement will not be considered a waiver of our right to enforce such provision. Our rights under this Agreement will survive any termination of this Agreement.
 
-#### 5. Complete Agreement
-These Terms of Service, together with the GitHub Privacy Statement, represent the complete and exclusive statement of the agreement between you and us. This Agreement supersedes any proposal or prior agreement oral or written, and any other communications between you and GitHub relating to the subject matter of these terms.
+#### 5. Amendments; Complete Agreement
+
+This Agreement may only be modified by a written amendment signed by an authorized representative of GitHub, or by the posting by GitHub of a revised version in accordance with [Section R. Changes to These Terms](/articles/github-terms-of-service/#r-changes-to-these-terms). These Terms of Service, together with the GitHub Privacy Statement, represent the complete and exclusive statement of the agreement between you and us. This Agreement supersedes any proposal or prior agreement oral or written, and any other communications between you and GitHub relating to the subject matter of these terms including any confidentiality or nondisclosure agreements.
```

Comment:
> I'm so surprised at all these requests for more capitalization.

I think you have a lot of developers who are used to writing case-sensitive code thinking "you didn't get that variable name right" ;).  It's also a nice "this word has a local definition" hint, and shows the writer intended to use that formal definition.  That's where I'm coming from, anyway.